test/recipe/Baked.Test.Recipe.Service/Baked.Test.Recipe.Service.csproj

```diff
@@ -5,12 +5,22 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\..\src\recipe\Baked.Recipe.Service\Baked.Recipe.Service.csproj" />
+    <Content Include="Core\PhysicalResource.txt">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
```

Comment:
physical resource should be in the `Application` project, not in the domain project